Assets/Tests/PlayMode/PlayerDataStorageManagerTests.cs

```diff
@@ -39,13 +39,9 @@ namespace PlayEveryWare.EpicOnlineServices.Tests.IntegrationTests
 
     public class PlayerDataStorageManagerTests : EOSTestBase
     {
-        EOSPlayerDataStorageManager playerDataStorageManager;
-
         [UnitySetUp]
         public IEnumerator UnitySetUp()
         {
-            playerDataStorageManager = new EOSPlayerDataStorageManager();
```

Comment:
We have the singleton, so we don't need to have this new-ing anymore. This comment is to observe that we do lose something, and gain something. Both of which are, there are no guarantees that the Manager will be entirely reset between runs.

This can be useful because if a test randomly breaks because of this new situation, then that's evidence we should fix something.

Reply:
This is a good point - but it is worth noting that the Services will _always_ be reset whenever the state of authentication changes - which should be the same thing. That is to say - the "state" of a service is not usually reset _except_ in circumstances where the currently authenticated user becomes no longer authenticated.